python/common/org/python/types/Int.java

```diff
@@ -929,4 +929,16 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
         }
         throw new org.python.exceptions.TypeError("'" + ndigits.typeName() + "' object cannot be interpreted as an integer");
     }
+
+
+    @org.python.Method(
+        __doc__ = "factorial(num)"
```

Comment:
The indentation for this code annotation is inconsistent with the code you added in _python/common/org/Python.java_ (for example line 1620). Our convention checklist states that we should use 4 spaces as indentation as you've done here, but it seems like this particular code block uses 8 spaces in the rest of the codebase (as you did in _python/common/org/Python.java_ ). It is difficult to judge which way is the correct one, but consistency is always good so perhaps pick one way and stick with it.

Reply:
We agree, we changed to 8 spaces since we now saw that they mostly used 8